cdap-support-bundle/src/main/java/io/cdap/cdap/support/handlers/SupportBundleHttpHandler.java

```diff
@@ -111,4 +135,91 @@ public void createSupportBundle(HttpRequest request, HttpResponder responder,
       responder.sendString(HttpResponseStatus.OK, prevInProgressUUID);
```

Comment:
This POST API is hard to use as it doesn't indicate if the ID returned is actually a still running one or a new one.